packages/create-atomic/template/start-netlify.mjs

```diff
@@ -0,0 +1,33 @@
+import getPort, {portNumbers} from 'get-port';
+import {spawn} from 'child_process';
+import {createRequire} from 'module';
+import fkill from 'fkill';
+
+const require = createRequire(import.meta.url);
+const args = process.argv.slice(2);
+
+const main = async () => {
+  // Dynamic Stencil port discovery feature, unsupported by Netlify.
+  const stencilPort = await getPort({port: portNumbers(3333, 3399)});
+
+  process.on('SIGINT', () => {
+    // Intermittent issue with Netlify not shutting down Stencil port properly.
```

Comment:
I would open an issue to Netlify about this but I'm not sure what isolates the issue for them to find fix it. Is it the OS, terminal, node version, Netlify or stencil version, something about Atomic itself? Lots of variables 🤷‍♂️